### ISSUE TITLE
color value of tokens are displayed in lowercase

### ISSUE THREAD

**Issue by tooomm** [Cockatrice/Cockatrice#1358]
Related to @ZeldaZach with his "Display color of cards" feature?
Or at least it made it visible maybe..?

All cards have uppercase values like **GU, R, BWG** etc.
Tokens have lowercase values like **g, wr** etc.

![colors for tokens in lowercase](https://cloud.githubusercontent.com/assets/9874850/9168456/0ea3944a-3f58-11e5-8616-c87afb039bf7.png)

**Comment by Fizztastic** [comment]
This happens because the entries in your tokens.xml are written in lower cap. Which also probably means that you have a custom/old tokens.xml file because the ones in https://github.com/Cockatrice/Magic-Token/blob/master/tokens.xml has it all written in upper caps like the cards.xml file generated by oracle.

**Comment by tooomm** [comment]
I ran oracle two weeks ago when I could access my pc after several weeks after installing first Lightning Angel version. I downloaded the tokens file twice over oracle in the last week. Last changes in the linked tokens file are older than that anyway...

But I could solve it:
All tokens from new `ORI` set are written in lower case in my token.xml, for example for  "Ashaya":
`<color>g</color>`. It's with capital letters in the source tokens file at git.

I used the Oracle shortcut in Cockatrice and ran the card and token update again. No change.
Checked tokens.xml it was now with capital `<color>G</color>`, but still displayed "g".
After restarting Cockatrice it was correct for all ORI tokens. Weird.

**Can be closed!**
